tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import annotations
+
+from pprint import pformat as pf
+from typing import Final
+
+import pytest
+from pytest import param
+
+from great_expectations import get_context
+from great_expectations.compatibility.sqlalchemy import inspect
+from great_expectations.data_context import EphemeralDataContext
+from great_expectations.datasource.fluent import (
+    PostgresDatasource,
+    SQLDatasource,
+)
+from great_expectations.expectations.expectation import (
+    ExpectationConfiguration,
+)
+
+PG_TABLE: Final[str] = "ge_expectations_store"
+TRINO_TABLE: Final[str] = "customer"
```

Comment:
This is dependent on some existing state.
Will be issuing a follow-up PR so these test setup the needed tables and cleanup.